Chp9Lsn11/UserInputHandler/UserInput.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UserInputHandler
+{
+    public class UserInput
+    {
+        static void Main(string[] args)
+        {
+
+        }
+
```

Comment:
All three input methods (UserString, UserArray, UserDoubles) are poorly named on two counts.  First, methods should be named as verbs or verb phrases.  This is because, in general, methods "do something" rather than "being something".  Second, regardless of what part of speech is used for naming, I can't tell what these methods do (unless I read the internal code carefully).  Maybe a name like "GetStringToReverse" or "PromptForStringToReverse".  Also, XML comments should be used.